[Email Routing] Updated docs to reflect new dash route (#5984)
cloudflare/cloudflare-docs: content/email-routing/setup/email-routing-dns-records.md

pcx_content_type: reference
weight: 2
meta:
    title: Email DNS records
---

# DNS records

You can check the status of your DNS records in the **Settings** section of Email Routing. This section also allows you to troubleshoot any potential problems you might have with DNS records.

## Email DNS records

> In the Email DNS records card you can check the status of your account's DNS records: 

Reviewer: ```suggestion
Check the status of your account's DNS records in the Email DNS records card:
```
Avoiding a repeated occurrence of "you can" (not sure if "Email DNS records" should appear in bold).

* **Email DNS records configured** - DNS records are properly configured.
* **Email DNS records misconfigured** - There is a problem with your accounts DNS records. Select **Enable Email Routing** to [start troubleshooting for problems](/email-routing/troubleshooting/).

### Start disabling

> When you successfully configure Email Routing, your DNS records will be locked and you will see a **Start disabling** button in the Email DNS records card. This is the recommended setting by Cloudflare. It means that the DNS records required for Email Routing to work are locked and can only be changed if you disable Email Routing on your domain.

Reviewer: ```suggestion
When you successfully configure Email Routing, your DNS records will be locked and the dashboard will show a **Start disabling** button in the Email DNS records card. This is the recommended setting by Cloudflare. It means that the DNS records required for Email Routing to work are locked and can only be changed if you disable Email Routing on your domain.
```
Avoiding "see".

If you need to delete Email Routing or migrate to another provider, select **Start disabling**. Refer to [Disable Email Routing](/email-routing/setup/disable-email-routing/) for more information.

### Lock DNS records

Depending on your zone configuration, you might have your DNS records unlocked. This will also be true if, for some reason, you have unlocked your DNS records. Select **Lock DNS records** to lock your DNS records and protect them from being accidentally changed or deleted. 

## View DNS records

> Select the **View DNS records** button for a list of the required `MX` and sender policy framework (SPF) records Email Routing is using. 

Reviewer: ```suggestion
Select **View DNS records** for a list of the required `MX` and sender policy framework (SPF) records Email Routing is using. 
```
